spk/rdiff-backup/src/installer.sh

```diff
@@ -24,14 +24,15 @@ postinst ()
     ${VIRTUALENV} ${INSTALL_DIR}/env > /dev/null
 
     # Install the wheels
-    ${INSTALL_DIR}/env/bin/pip install --no-deps --no-index -U --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse ${INSTALL_DIR}/share/wheelhouse/*.whl > /dev/null 2>&1
+    ${INSTALL_DIR}/env/bin/pip3 install --no-deps --no-index -U --force-reinstall -f ${INSTALL_DIR}/share/wheelhouse ${INSTALL_DIR}/share/wheelhouse/*.whl > /dev/null 2>&1
 
     # Fix shebang
-    sed -i -e "s|^#!.*$|#!${INSTALL_DIR}/env/bin/python|g" ${INSTALL_DIR}/env/bin/rdiff-backup
+    sed -i -e "s|^#!.*$|#!${INSTALL_DIR}/env/bin/python3|g" ${INSTALL_DIR}/env/bin/rdiff-backup
 
     # Add symlink
     mkdir -p /usr/local/bin
     ln -s ${INSTALL_DIR}/env/bin/rdiff-backup /usr/local/bin/rdiff-backup
+    ln -s ${INSTALL_DIR}/env/bin/rdiff-backup-statistics /usr/local/bin/rdiff-backup-statistics
```

Comment:
Remarks while implementing generic installer for rdiff-backup_
- `SPK_COMMANDS` is only supported when not using `INSTALLER_SCRIPT` nor `SERVICE_SETUP`. For wheel installation we need at least a custom SERVICE_SETUP script (tell me if I am wrong) and therefore the generic link creation with SPK_COMMANDS is not supported (I added a related comment to the Makefile variables wiki-page some days ago).
- All cross compiled wheels are for x86_64 arch and cannot be installed on non x64 targets (I think this is under your scope with #3944).
- with the generic start-stop-status script the status of the package is always running, even the installation failed (post installation of the wheels). We should find a generic way to handle errors in post-installation to avoid a running status (avoid the need of a custom start-stop-status script just for the sake of error handling).